documentation/asciidoc/computers/linux_kernel/building.adoc

```diff
@@ -106,6 +114,18 @@ sudo cp arch/arm/boot/dts/overlays/README /boot/overlays/
 sudo cp arch/arm/boot/zImage /boot/$KERNEL.img
 ----
 
+For the 64-bit kernel:
+
+[,bash]
+----
+make -j4 Image modules dtbs
+sudo make modules_install
+sudo cp arch/arm64/boot/dts/*.dtb /boot/
+sudo cp arch/arm64/boot/dts/overlays/*.dtb* /boot/overlays/
+sudo cp arch/arm64/boot/dts/overlays/README /boot/overlays/
+sudo cp arch/arm64/boot/zImage /boot/$KERNEL.img
```

Comment:
Comparing this to the cross-compile instructions, should the `zImage` here be `Image` ?
(which would also seem to match up with the `make -j4 Image modules dtbs` line above?)
ping @ghollingworth

Reply:
Yes, I missed that one...  As I said I still could do with someone (me) going through the instructions again just to make sure my changes are valid.  Unfortunately I don't have a 64-bit version running at home at the moment and would need to install a new one...  May be able to do that tomorrow morning.